README.md

```diff
@@ -10,6 +10,11 @@ output can sometimes be post-processed.
 
 Each recipe should try to answer a single question.
 
+# Software requirements
+
+- You will need python 3.6 and pip for python 3.6 installed on your machine to run the program.
```

Comment:
Good call! Though in Python 3, pip comes bundled with python so there should be no need to install it separately. Also please change "python 3.6" to "Python 3.6 or higher"

Reply:
Oh great, yep!